src/components/new-request/search.vue

```diff
@@ -65,7 +100,7 @@
                 @change="setClearErrors()"
                 hide-details="auto"
                 filled
-                v-model="entity_type_cd"
+                v-model="entity_type_options_select_bind"
```

Comment:
I got a rendering issue if binding `entity_type_cd` for alteration because it changed as we chose different alterations. Current solution for it is to change it to `origin_entity_type_cd` for alteration only.

Reply:
Let's go with this solution for now.

Could you add a comment on line 516 explaining why it's different for conversion? Thanks.